src/chi-vue/src/constants/events.ts

```diff
@@ -18,13 +18,13 @@ export const DATA_TABLE_EVENTS = {
   },
 };
 export interface DataTableSorting {
-  column: string;
-  direction: 'ascending' | 'descending';
-  data: DataTableRow[] | undefined;
+  column: string | undefined;
+  direction: 'ascending' | 'descending' | undefined;
```

Comment:
Not in this case as undefined column / direction is necessary for the case of the 3rd click (1st click: ascending, 2nd click: descending, 3rd click: no sorting scenario). This way, on `No Sorting`scenario developers receive the following data `{ column: undefined, direction: undefined, data: undefined }`
 Otherwise, `chiDataSorting`would emit without any data.

Reply:
You're right. Thank you Jani.